.editorconfig

```diff
@@ -13,3 +13,5 @@ trim_trailing_whitespace = false
 [{.travis.yml,npm-shrinkwrap.json,package.json}]
 indent_style = space
 indent_size = 2
+[Makefile]
+indent_style = tab
```

Comment:
@SISheogorath Thanks for telling more.
But I think it won't apply to Makefile, because "a tab" indent is part for the format.